flink-runtime/src/main/java/org/apache/flink/runtime/io/network/partition/hybrid/HsConsumerId.java

```diff
@@ -0,0 +1,64 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.runtime.io.network.partition.hybrid;
+
+import java.util.Objects;
+
+/** This class represents the identifier of hybrid shuffle's consumer. */
+public class HsConsumerId {
+    /**
+     * This consumer id is used in the scenarios that information related to specific consumer needs
+     * to be ignored.
+     */
+    public static final HsConsumerId ANY_CONSUMER_ID = new HsConsumerId(-1);
+
+    /**
+     * This consumer id is used in the scenarios that only one consumer is allowed for a single
+     * subpartition.
+     */
+    public static final HsConsumerId SINGLE_CONSUMER_ID = new HsConsumerId(0);
```

Comment:
I'd suggest to name this `DEFAULT`.
- It is guaranteed that if there's only one consumer, this id will be used. However, there's no guarantee for the other way. It is possible that this id is used when there're multiple consumers.